doc/nrf/ug_chip.rst

```diff
@@ -46,6 +46,32 @@ The |NCS|'s Multiprotocol Service Layer (MPSL) driver allows running Bluetooth L
 
 For detailed description, see the `nRF Connect platform overview`_ CHIP documentation page.
 
+Configuration in |NCS|
+**********************
+
+The CHIP nRF Connect platform is built when programming any of the available :ref:`chip_samples` to the supported development kits, which creates a CHIP accessory device.
+Such a device requires additional configuration if it is to be paired and controlled remotely over a CHIP network built on top of a low-power, 802.15.4 Thread network.
+The following figure shows an overview of the network topology with the required configuration elements.
+
+.. figure:: images/CHIP_configuration_overview.jpg
+   :scale: 50 %
+   :alt: CHIP network topology
+
+   CHIP network topology
+
+For information about how to configure the required components, see the following documentation:
+
+* For configuring Wi-Fi Access Point and Thread Border Router:
+
+  * `Building and programming OpenThread RCP firmware`_ - This will create the OpenThread RCP device required for forming the Thread network.
+  * `Configuring PC as Thread Border Router`_
+
+* For setting up CHIP Controller and Bluetooth LE Commissioning: `Building and installing Android CHIPTool`_
+* For preparing the accessory device and commissioning it: `Preparing and commissioning accessory device`_
+
+You can find this information in the CHIP sample documentation as well.
+If you are new to CHIP, check also the complete getting started tutorial `CHIP early development demo on nRF52840`_ on DevZone.
```

Comment:
This sentence may suggest that NCS documentation is not "complete". Also, I am a bit worried that someone will try to install a mobile phone application from the blog post and try to pair it with CHIP accessory from NCS - this is no longer compatible.

Thinking about this more.. I would say it is important to have the same set of the information (and now we have it - thanks!) as in blogpost and not necessary link to it as it was part of the NCS documentation.

So either we should rephrase the last sentence, move it to a different section (e.g. links), or completely remove.

What do you think @greg-fer @Damian-Nordic @kkasperczyk-no?

Reply:
Replaced with a link to DevZone instead.